sdk/storage/Microsoft.Azure.WebJobs.Host.EndToEndTests/tests/BlobTriggerEndToEndTests.cs

```diff
@@ -166,12 +167,12 @@ public void BlobChainStepThree([QueueTrigger(BlobChainCommittedQueueName)] strin
             }
         }
 
-        [WebJobsLiveOnly]
+        [LiveOnly]
         [TestCase("AzureWebJobsSecondaryStorage")]
         [TestCase("AzureWebJobsStorage")]
         public async Task PoisonMessage_CreatedInCorrectStorageAccount(string storageAccountSetting)
         {
-            var storageAccount = StorageAccount.NewFromConnectionString(Environment.GetEnvironmentVariable(storageAccountSetting));
+            var storageAccount = StorageAccount.NewFromConnectionString(GetVariable(storageAccountSetting.ToUpperInvariant()));
```

Comment:
Define this as a property of TestEnvironemtn see https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/tests/TablesTestEnvironment.cs#L16 as an example

Reply:
ok.